buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -148,6 +150,9 @@ required for Mantid development.
 
 %changelog
 
+* Mon Jan 22 2017 Martyn Gigg <martyn.gigg@stfc.ac.uk>
```

Comment:
Doesn't build because of wrong year. It is now 2018.

Reply:
:man_facepalming: